internal/controllers/manager.go

```diff
@@ -51,9 +50,11 @@ func (m *TiltServerControllerManager) GetClient() ctrlclient.Client {
 func (m *TiltServerControllerManager) SetUp(ctx context.Context, st store.RStore) error {
 	ctx, m.cancel = context.WithCancel(ctx)
 
-	// TODO(milas): we should provide a logr.Logger facade for our logger rather than using zap
-	w := logger.Get(ctx).Writer(logger.DebugLvl)
-	ctrl.SetLogger(zap.New(zap.UseDevMode(true), zap.WriteTo(w)))
+	// controller-runtime internals don't really make use of verbosity levels, so in lieu of a better
```

Comment:
of a better what 👀

Reply:
🤦 One day I will re-wrap a comment without losing a